awx/main/conf.py

```diff
@@ -795,6 +795,16 @@
     category_slug='bulk',
 )
 
+register(
+    'UI_NEXT',
+    field_class=fields.BooleanField,
+    default=False,
+    label=_('Enable Next Generation User Interface'),
+    help_text=_('Enable the next generation user interface.'),
```

Comment:
optional suggestion:

"next generation" has other colloquial meanings, for example to indicate "cutting edge"

maybe we can consider "Enable experimental user interface"

Reply:
or "tech preview" as indicated elsewhere

Reply:
updated